pkg/systemd/notifyproxy/notifyproxy.go

```diff
@@ -189,7 +188,10 @@ func (p *NotifyProxy) SocketPath() string {
 // Close closes the listener and removes the socket.
 func (p *NotifyProxy) Close() error {
 	defer os.Remove(p.socketPath)
-	return p.connection.Close()
+	if err := p.connection.Close(); err != nil {
+		return err
+	}
+	return nil
```

Comment:
this change is not needed, the if branch can be removed.

Reply:
:+1: Will fix after rebase for green CI